authenticate/handlers.go

```diff
@@ -24,10 +24,10 @@ import (
 // CSPHeaders are the content security headers added to the service's handlers
 // https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src
 var CSPHeaders = map[string]string{
-	"Content-Security-Policy": "default-src 'none'; style-src 'self'" +
-		" 'sha256-z9MsgkMbQjRSLxzAfN55jB3a9pP0PQ4OHFH8b4iDP6s=' " +
-		" 'sha256-qnVkQSG7pWu17hBhIw0kCpfEB3XGvt0mNRa6+uM6OUU=' " +
-		" 'sha256-qOdRsNZhtR+htazbcy7guQl3Cn1cqOw1FcE4d3llae0='; " +
+	"Content-Security-Policy": "default-src 'none'; style-src " +
+		"'sha256-spMkVDoBBY86p0RC1fBYwdnGyMypJM8eG57+p3VASyk=' " +
```

Comment:
CSS is currently broken on authenticate because of this :(

Reply:
Any way we can generate this automatically as part of the build?  This will get broken again.

Reply:
Let me pull this out into a new PR. I have an idea of how we can avoid inlining, and avoid this more or less all-together.

Reply:
Addressed in #392